test/functionalTest/cases/1156_mqfilters_and_compounds/mqfilters_and_compounds_existence_subs.test

```diff
@@ -46,7 +45,8 @@ accumulatorStart --pretty-print
 # 12. Create subscription 6 for mq=!A1.M2.i - notification
 # 13. Dump and reset accumulator, see one notification
 # 14. Update E1, to provoke another 3 notifications
-# 15. Dump and reset accumulator, see three notifications
+# 15. Dump and reset accumulator, see three notifications (sub1, sub4 and sub6)
```

Comment:
The modifications in this .test are related with "Fix: wrong matching in metadata existence and not existence filters with compounds" fix.

It could seem that the .test has not been actually notified. However, up to know it was wrong: notifications in step 15 were for sub1 (ok), sub2 (wrong) and sub6 (ok). Now the outcome is corret and step 16 has been added to ensure that.

NTC (just informative)